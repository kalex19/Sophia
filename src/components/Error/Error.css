.error-container {
  margin: 5% 35%;
}

.error-message {
  color: white;
  font-size: 4em;
  font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
}

.home-button {
  height: 60px;
  width: 220px;
  color: darkgray;
  font-size: 2em;
  margin: 0 15%;
  box-shadow: 5px 5px;
  border-radius: 50%;
  outline: none;
}

.home-button:hover {
  color: rgb(14, 107, 14);
}