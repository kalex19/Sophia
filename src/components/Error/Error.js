import React from 'react';
import './Error.css';
import { Link } from 'react-router-dom';

export const Error = () => {
    return (
      <div className="error-container">
        <h2 className="error-message">Page Not Found</h2>
        <Link to={'/'}>
        <button className="home-button">Return Home</button>
				</Link>
      </div>
    )
  };

export default Error;
