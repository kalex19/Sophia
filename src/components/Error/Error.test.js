import React from 'react';
import Error from './Error';
import {shallow} from 'enzyme';

describe('Error', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<Error/>)
  });

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})