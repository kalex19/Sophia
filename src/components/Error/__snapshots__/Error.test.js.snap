// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error should match snapshot 1`] = `ShallowWrapper {}`;
